lib/sidekiq_prometheus.rb

```diff
@@ -214,6 +214,19 @@ def sidekiq_setup
     end
   end
 
+  # Attempt to load Rackup to see if we're in Rack > 3.0
+  # This allows us to support both Rack 2.x and 3.x
+  def webrick_handler
+    @_webrick_handler ||= begin
+      begin
+        require 'rackup'
```

Comment:
fixup for standardrb:

```suggestion
        require "rackup"
```